packages/wouter-preact/types/use-location.d.ts

```diff
@@ -39,10 +39,11 @@ export const useSearch: () => string;
 
 export const usePathname: (options?: { ssrPath?: Path }) => Path;
 
-export const navigate: (
-  to: string | URL,
-  options?: { replace?: boolean }
-) => void;
+export const useHistoryState: <T = any>() => T;
```

Comment:
I think we can rename this file to `use-browser-location`, as well as move everything that goes before `export type Primitive` to `index.d.ts`

Reply:
I renamed the file.

but moving is not great because it is harder to maintain `preact` types